modules/lang-painless/build.gradle

```diff
@@ -28,6 +28,15 @@ integTestCluster {
   module project.project(':modules:mapper-extras')
 }
 
+Task additionalClusterTest = tasks.create(
+        name: "additionalClusterTest", type: RestIntegTestTask)
```

Comment:
Can we use a less generic name? Something specific to testing doc values missing values?